test/board_test.rb

```diff
@@ -0,0 +1,114 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/board'
+require './lib/cell'
+require './lib/ship'
+
+class BoardTest < Minitest::Test
+
+  def setup
+    @board = Board.new
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+  end
+
+  def test_it_exists
+    assert_instance_of Board, @board
+  end
+
+  def test_cells_returns_a_hash
+    assert_instance_of Hash, @board.cells
+  end
+
+  def test_16_elements_in_hash
```

Comment:
This is not proper way to test our hash. We need to re-evaluate how to test each element.